scripts/release/go_update/run_GO_Update.pl

```diff
@@ -0,0 +1,92 @@
+#!/usr/local/bin/perl  -w
+use strict;
+
+use lib '/usr/local/gkb/modules';
+use GKB::Config;
+
+use autodie qw/:all/;
+use Cwd;
+use Getopt::Long;
+use Data::Dumper;
+use File::Basename;
+use List::MoreUtils qw/uniq/;
+
+use Log::Log4perl qw/get_logger/;
+
+Log::Log4perl->init(\$LOG_CONF);
+my $logger = get_logger(__PACKAGE__);
+
+my $data_release_pipeline_application_version = "1.0.0";
+my $data_release_pipeline_application = "go-update";
+my $data_release_pipeline_tag = "go-update-1.0.0";
+my $data_release_pipeline_repository = "https://github.com/reactome/data-release-pipeline";
+my $resource_dir = "data-release-pipeline/".$data_release_pipeline_application."/src/main/resources/";
+my $start_directory = cwd(); # abs_path($0);
+
+my $release_version = @args[0];
+
+print "start_directory: ".$start_directory."\n";
+
+if (-e "data-release-pipeline") {
+  $logger->info("data-release-pipeline exists, pulling");
+  # Start on develop. Pull will fail if we're not already on a branch
+  chdir "data-release-pipeline";
+  system("git checkout develop");
+  if (!$data_release_pipeline_tag) {
+    system("git pull");
+  # if there is a tag defined, we should check that out.
+  } else {
+    #chdir "data-release-pipeline";
+    # Still need to do a pull, since the tag might be been created after the last pull
+    system("git pull");
+    system("git checkout ".$data_release_pipeline_tag);
+    #chdir "..";
+  }
```

Comment:
The `system("git pull")` can be done before the condition since it occurs in the if and the else.  The condition could then be just:
```
if ($data_release_pipeline) {
    system("git checkout $data_release_pipeline_tag");
}
```